### DISCUSSION TITLE
Add achivements other than ranking up

### DISCUSSION THREAD

**Discussion by lulunac27a** [codewars/codewars.com#3273] [Ideas]
I want to see the achievements like reaching a number of Katas has been completed, reaching a number of Katas completed in a day, reaching a specified number of daily streaks, and more. Getting these achievements listed aboue should earn Honor points and place it in the Achievements category in the Honor Breakdown section of the user Stats section.

**Comment by hobovsky** [comment]
Seems to be a duplicate of https://github.com/codewars/codewars.com/issues/604
